src/Resources/config/twig.xml

```diff
@@ -36,5 +36,13 @@
             <argument type="service" id="sonata.admin.global_template_registry"/>
             <argument type="service" id="service_container"/>
         </service>
+        <service id="sonata.templates.twig.environment" class="Core23\AdminBundle\Twig\AdminEnvironment" public="true">
```

Comment:
Core23 huh ? ^^

Reply:
WIP. Just copy & paste of my local stuff 😄